docs/user-guide/installation/mac.md

```diff
@@ -2,11 +2,11 @@
 
 ## Requirements
 
-- C++11 compiler (C++14 preferred).
-- CMake 3.4 or newer
+- C++17 compiler (C++14 preferred).
```

Comment:
```suggestion
- C++14 compiler (C++17 preferred).
```

Should this be C++14 compiler, with 17 preferred?

Reply:
this should be just a C++17 compiler

Reply:
```suggestion
- C++17 compiler.
```